src/ontology/genepio-edit.owl

```diff
@@ -4370,7 +4370,7 @@ AnnotationAssertion(obo:GENEPIO_0000001 obo:GENEPIO_0000023 "Collection")
 AnnotationAssertion(obo:GENEPIO_0000004 obo:GENEPIO_0000023 "Received date")
 AnnotationAssertion(obo:GENEPIO_0000005 obo:GENEPIO_0000023 "NML Labware LIMS")
 AnnotationAssertion(obo:IAO_0000112 obo:GENEPIO_0000023 "year/month/day")
-AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000023 "date lab received isolate")
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0000023 "A date which describes when the laboratory received the isolate."@en)
```

Comment:
Too picky? "A date which describes when **a** laboratory received **an** isolate."

Reply:
Nope that definitely sounds better :)